website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables.md

```diff
@@ -3,23 +3,79 @@ title: Environment variables
+
+ - Default argument supplied to the `env_var` Jinja function in code
+ - Project level default value
+ - Environment level values
+ - Personal and job override values
+    - Each developer can set a personal value to override values set in the developent environment 
+    - Job level values can override values set in deployment environments
+
+**Setting environment variables at the project and environment level**
+
+To set environment variables at the project and environment level, head over to the Environments section in your main navigation pane. You'll see a view to list all your environments and a view to set and update environment variables. Make sure you're in environment variable view.
+
+[TO CREATE GIF]
+
+You'll notice there is a `Project Default` column. This is a great place to set a value that will persist across your whole project, independent of where the code is run. We recommend setting this value when you want to supply a catch-all default or add a project-wide token or secret.
+
+
+To the right of the `Project Default` column are all your environments. Values set at the environment level take priority over the project level default value. This is where you can tell dbt Cloud to intepret an environment value differently in your Staging vs. Production environment, as example.
+
+**Overridng environment variables at the job level**
+
+You may have multiple jobs that run in the same environment, and you'd like the environment variable to be interpretted differently depending on the job. 
+
+When setting up or editing a job, you will see a section for overriding environment variables. Every job is specified to run in a specific, deployment environment, and by default, a job will inherit the values set at the environment level for the environment in which it runs. If you'd like to set a different value at the job level, edit the value to override it.
+
+[TO CREATE GIF]
+
+
+**Overridng environment variables at the personal level**
+
+Developers may also want to set a personal value override for an environment variable while developing in the dbt IDE. By default, dbt Cloud will look to use environment variable values set in the project's development environment. To see and override these values, developers should head to their Profile and then select their project under Credentials and scroll to the Environment Variables section. 
+
+[TO CREATE GIF]
+
+
+To supply an override, developers can edit and specify a different value to use. These values will be respected in the IDE both for the Results and Compiled SQL tabs.
+
+[TO CREATE GIF]
+
+:::info Appropriate coverage
+If you have not set a project level default value for every environment variable, it may be possible that dbt Cloud does not know how to interpret the value of an environment variable in all contexts. We leave it to our users to either supply a default or make sure all scenarios where dbt runs have been covered; otherwise, dbt will throw a compile error.
```

Comment:
```suggestion
If you have not set a project level default value for every environment variable, it may be possible that dbt Cloud does not know how to interpret the value of an environment variable in all contexts. You should either supply a default value or ensure that specific environment variables are specified for all relevant environments; otherwise, dbt will throw a compile error.
```

Reply:
I don't feel *incredibly* attached to this specific wording, but I think we should avoid talking about "we" as the people who make dbt Cloud and we should definitely avoid using the possessive "we" when talking about dbt Cloud users.